skglm/estimators.py

```diff
@@ -423,7 +350,7 @@ def fit(self, X, y):
         self :
             Fitted estimator.
         """
-        return _glm_fit(X, y, self, Quadratic(), L1(self.alpha))
+        return self.solver.solve(X, y, Quadratic(), L1(self.alpha))
 
     def path(self, X, y, alphas, coef_init=None, return_n_iter=True, **params):
```

Comment:
we want to keep this (for compatibility with sklearn) but I think the `cd_solver_path` import is gone

We should replace it by solver.path here